lib/Page.js

```diff
@@ -823,10 +823,18 @@ class Page extends EventEmitter {
     return this.mainFrame().title();
   }
 
-  async close() {
+  /**
+   * @param {!{runBeforeUnload: boolean}} options
```

Comment:
add a ?, so runBeforeUnload?: boolean, to indicate an optional property.

Reply:
I did the `(boolean|undefined)` to be consistent with other parts of our codebase regarding this.